src/inline.d

```diff
@@ -1619,17 +1620,26 @@ public:
         {
             printf("FuncDeclaration::inlineScan('%s')\n", fd.toPrettyChars());
         }
-        if (fd.isUnitTestDeclaration() && !global.params.useUnitTests)
+        if (fd.isUnitTestDeclaration() && !global.params.useUnitTests ||
+            fd.flags & FUNCFLAGinlineScanned)
```

Comment:
We have unparenthesized &&, ||, and & - that's bound to confuse folks. How about some good parens around the latter?